
wchar.h must be included directly (and not via _G_config.h, which happens eg.
when including stdio.h before wchar.h). strange - can't reproduce it in the
chroot filesystem.  ~~ Clifford

--- ./ncurses/curses.priv.h	2007-11-19 20:35:11.000000000 +0000
+++ ./ncurses/curses.priv.h	2007-11-19 20:39:07.000000000 +0000
@@ -32,6 +32,8 @@
  *     and: Thomas E. Dickey                        1996-on                 *
  ****************************************************************************/
 
+#include <wchar.h>
+
 
 /*
  * $Id: curses.priv.h,v 1.314 2006/12/10 00:55:14 tom Exp $
